public/js/entry.js

```diff
@@ -1,5 +1,5 @@
 angular.module('main')
-  .controller('EntryCntrl', ['$scope', '$firebase', 'fbTop', function($scope, $firebase, fbTop){
```

Comment:
This I think is hilarious.  Does it really make any kind of difference that I have an n in my controllers?